src/worker.js

```diff
@@ -96,9 +98,6 @@ waitUntilGraphIsReady()
         cron: CRON_NIGHTLY,
       },
     });
-
-    // Always write edges.json file on start to make sure it exists
-    submitJob(tasks.exportEdges, 'exportEdges-initial');
```

Comment:
Why did you delete this? I think it's important.

Reply:
because now with our new approach, the file is created on the first round so no need to queue task but we can leave it  just to be sure

Reply:
When do you say it's created in the first iteration?
The exportEdges task in only triggered after a trust or a transfer. If there's no such events the exportEdges doesn't happen. Thus we need the initial export.

Reply:
agreed, I added it again as it was